src/pybind/mgr/volumes/fs/operations/group.py

```diff
@@ -181,10 +185,12 @@ def open_group(fs, vol_spec, groupname):
             raise VolumeException(-e.args[0], e.args[1])
     yield group
 
+
 @contextmanager
 def open_group_unique(fs, vol_spec, groupname, c_group, c_groupname):
     if groupname == c_groupname:
         yield c_group
     else:
         with open_group(fs, vol_spec, groupname) as group:
             yield group
+
```

Comment:
Remove this extra line. See [this](https://www.flake8rules.com/rules/E302.html) to understand why the extra blank line is required.